lib/Partitioner/Partitioner.cpp

```diff
@@ -93,19 +101,46 @@ void Partitioner::dumpDAG(llvm::StringRef dotFilename) const {
   return;
 }
 
+llvm::Error
+Partitioner::logicalDevicesValidation(NodeToFunctionMap &partitions) {
```

Comment:
Do you have a Doxygen comment for this function anywhere? If not, please add it.

Reply:
Yes. In Partitioner.h.